tests/input/BINARY_OP.py

```diff
@@ -0,0 +1,25 @@
+(a, b) = (10, 3)
+print('Addition:', a + b)
+print('Subtraction:', a - b)
+print('Multiplication:', a * b)
+print('Division:', a / b)
+print('Modulo:', a % b)
+print('Floor Division:', a // b)
+print('Exponentiation:', a ** b)
+print('Greater than:', a > b)
+print('Less than:', a < b)
+print('Greater than or equal to:', a >= b)
+print('Less than or equal to:', a <= b)
+print('Equal to:', a == b)
+print('Not equal to:', a != b)
+print('Logical AND:', True and False)
+print('Logical OR:', True or False)

```

Comment:
To avoid constant folding, try
```suggestion
print('Logical AND:', a and b)
print('Logical OR:', a or b)
```